PWGHF/Tasks/taskMcValidation.cxx

```diff
@@ -515,6 +539,8 @@ struct HfTaskMcValidationRec {
      {"TrackToCollChecks/histAmbiguousTrackZvtxRMS", "RMS of #it{Z}^{reco} of collisions associated to a track;RMS(#it{Z}^{reco}) (cm);entries", {HistType::kTH1F, {{100, 0., 0.5}}}},
      {"TrackToCollChecks/histFracGoodContributors", "Fraction of PV contributors originating from the correct collision;fraction;entries", {HistType::kTH1F, {{101, 0., 1.01}}}},
      {"TrackToCollChecks/histCollisionsSameBC", "Collisions in same BC;number of contributors collision 1;number of contributors collision 2;#it{R}_{xy} collision 1 (cm);#it{R}_{xy} collision 2 (cm);number of contributors from beauty collision 1;number of contributors from beauty collision 2;", {HistType::kTHnSparseF, {axisMult, axisMult, axisR, axisR, axisSmallNum, axisSmallNum}}}}};
+  HistogramRegistry registryMesons{"registryMesons"};
+  HistogramRegistry registryBaryons{"registryBaryons"};
```

Comment:
@Luca610 I didn't want to block your PR from merging, but please note that using multiple histogram registries (differing only in names) has no effect on how your histograms are stored. Therefore declaring several just adds unnecessary complexity in the code and useless struct members.

Reply:
Hi @vkucera, sorry for my late reply. I added a second registry because apparently there is a limit on the number of histograms that can be stored in a single registry. The previos version of the task was barely below that limit and I recieved a compilation error after introducing the resonances. I knw the solution of two registries is far from ideal...

Reply:
Thanks for the explanation. I didn't know there was a limit to the number of histograms in a histogram registry. Just for my education, what is the limit?

Reply:
It is 512.